### DISCUSSION TITLE
Show notification on validation error

### DISCUSSION THREAD

**Discussion by supermx1** [directus/directus#7035] [Feature Requests]
### Describe the Bug

Okay, so if you create a collection with a number of fields, maybe about 20 fields, and some are required fields (Doesn't allow null). When filling the form and you miss a required field and hit the submit button it doesn't scroll to the fields missing a value... The checkmark just spins and stops, there are no error or validation notifications. You then have to start scrolling through the form to find the empty field. 

Suggestion
The pop-up that appears at the bottom right corner of the screen when an item is saved successfully can also be triggered during a validation error, showing the number of fields (field name/translation name inclusive). So the user can identify and scroll to the empty field directly, instead of having to look for the highlighted fields individually.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
